src/Tribe/Tickets.php

```diff
@@ -2668,17 +2669,66 @@ public function get_ticket_prices( array $prices, $post_id ) {
 				if ( in_array( $ticket->price, $prices ) ) {
 					continue;
 				}
-
-
 				// An empty price property can be ignored (but do add if the price is explicitly set to zero)
-				elseif ( isset( $ticket->price ) && is_numeric( $ticket->price ) ) {
+				if ( isset( $ticket->price ) && is_numeric( $ticket->price ) ) {
 					$prices[] = $ticket->price;
 				}
 			}
 
 			return $prices;
 		}
 
+		/**
+		 * Filter past tickets from showing up in cost range.
+		 *
+		 * @since TBD
+		 *
+		 * @param array  $costs List of ticket costs.
+		 * @param int    $post_id Target Event's ID.
+		 * @param string $meta Meta key name.
+		 * @param bool   $single determines if the requested meta should be a single item or an array of items.
+		 *
+		 * @return mixed
+		 */
+		public function exclude_past_tickets_from_cost_range( $costs, $post_id, $meta, $single ) {
+
+			if ( '_EventCost' != $meta || $single || empty( $costs )  ) {
+				return $costs;
+			}
+
+			$exclude_past_tickets = apply_filters( 'event_tickets_exclude_past_tickets_from_cost_range', true );
```

Comment:
```suggestion
			/**
			 * Allow filtering of whether to exclude past tickets in the event cost range.
			 *
			 * @since TBD
			 *
			 * @param bool $exclude_past_tickets Whether to exclude past tickets in the event cost range.
			 */
			$exclude_past_tickets = apply_filters( 'event_tickets_exclude_past_tickets_from_cost_range', true );
```